mobile/src/modules/transactions/components/send/SendBurstForm.tsx

```diff
@@ -102,19 +103,36 @@ const styles = StyleSheet.create({
 
 export class SendBurstForm extends React.Component<Props, SendBurstFormState> {
 
+
+  getAccounts = (): Array<SelectItem<string>> => {
+    return this.props.accounts
+      .filter(({ keys }) => keys && keys.publicKey)
+      .map((account) => ({
+        value: account.accountRS,
+        label: `...${last(account.accountRS.split('-'))}`
+      }));
+  }
+
+  getAccount = (address: string): Account | null => {
+    return this.props.accounts
+      .find(({ accountRS }) => accountRS === address) || null;
+  }
+
   setupState = (props?: SendBurstFormState) => {
+    const accounts = this.getAccounts();
     return {
-      sender: null,
+      sender: accounts.length === 1 ? this.getAccount(accounts[0].value) : null,
       amount: props && props.amount || '',
       fee: props && props.fee ||
            this.props.suggestedFees &&
            convertNQTStringToNumber(this.props.suggestedFees.standard.toString()).toString() || '',
       message: props && props.message || undefined,
-      messageIsText: props && props.messageIsText || true,
+      messageIsText: props && typeof props.messageIsText !== 'undefined' ? props.messageIsText : true,
```

Comment:
does this fix the issue with non-Text messages? Oh, I think it was fixed already, or not?